### ISSUE TITLE
(1.7.10) poof! crashes.

### ISSUE THREAD

**Issue by Migmag789** [iopleke/Minechem#845]
Hi ! I have a crash when I launch a map with your mods ;) Please can you fix the crash ? :D

**Comment by chaos234** [comment]
Without any Crash Report nothing can be fixed. This should be added by default because ... logic!

**Comment by Migmag789** [comment]
Yes I know, but there isn't any crash report :/ That's when my world load, the only thing I know that's caused by Better Builder Wand AND Minechem both :/
If I remove Minechem that works, if I remove BBW that works ;)
